dataland-keycloak/dataland_theme/login/resources/scss/main.scss

```diff
@@ -1,11 +1,11 @@
-@import 'theme';
-@import 'global';
-@import 'text-inputs';
-@import 'checkbox';
-@import 'separator';
-@import 'modals';
-@import 'social-login';
-@import 'two-factor-authentication';
-@import 'ibm-plex-sans';
-@import 'password-strength';
-@import 'link-accounts';
+@use 'theme';
```

Comment:
In diesem Teil ist auch wieder das Problem das auf Arial zurückgefallen wird. Wenn man die Keycloak login page aufmacht sieht man das sehr gut.